spec.bs

```diff
@@ -849,6 +857,11 @@ a reporting destination=] through the reference it kept, it will handle all of t
 stored in the [=list=]. If the destination is never finalized, then the pending events will never
 be sent.
 
+An <dfn for=fencedframetype>untrusted network status</dfn> is either "<dfn export for='untrusted
+network status'>`enabled`</dfn>", "<dfn export for='untrusted network status'>`disabled for this
+tree`</dfn>", or "<dfn export for='untrusted network status'>`disabled for this and descendant
```

Comment:
How about the following renames:

 - "disabled for this tree" --> "disabled for this frame". Or is there another reason "tree" was chosen? My understanding is that this is an intermediate state where network is revoked just for this frame, but not for any subframes fenced/unfenced.
 - "disabled for this and descendant tree" --> "disabled for full subtree" (means that all frames, fenced/unfenced underneath the one in question, has network disabled)

What do you think?

----

Separately, I'd appreciate you adding a note underneath indicating that the middle state is not a final state; it is just an intermediate/limbo state that we're in until .... (non-normatively enumerate the conditions under which a frame has it switch from this middle state to the state).

Reply:
"tree" was chosen simply because that's the naming convention we used in the implementation, which is scoped to fenced frame trees.

> My understanding is that this is an intermediate state where network is revoked just for this frame, but not for any subframes fenced/unfenced.

A child iframe will also have its network access revoked at the intermediate stage, so saying "disabled for this frame" would not be correct.

Would "disabled for this tree" and "disabled for all subtrees" be a better naming convention? Subtrees here is referring to any descendant fenced frame trees.

I've also added a note outlining what the states mean.

Reply:
If "subtree" is really just a synonym for "unfenced", then I think the following names would make more sense "disable for this tree" and "disabled for this tree and fenced subtrees"

Reply:
Replaced the names with "disabled for this tree" and "disabled for this tree and fenced subtrees".